tests/utils/compute_groundtruth.cpp

```diff
@@ -278,8 +279,7 @@ inline void load_bin_as_float(const char *filename, float *&data, size_t &npts_u
   uint64_t end_id = (std::min)(start_id + PARTSIZE, (uint64_t) npts_i32);
   npts_u64 = (uint64_t) end_id - (uint64_t) start_id;
```

Comment:
(From the previous PR) Are the two casts here necessary, the two variables already have `uint64_t` type?